vclib-aries/src/commonTest/kotlin/at/asitplus/wallet/lib/aries/DummyCredentialDataProvider.kt

```diff
@@ -30,7 +31,8 @@ class DummyCredentialDataProvider(
         if (credentialScheme != ConstantIndex.AtomicAttribute2023) {
             return KmmResult.failure(UnsupportedOperationException("no data"))
         }
-        val subjectId = subjectPublicKey.toJsonWebKey().identifier
+        val subjectId = subjectPublicKey.toJsonWebKey()
+            .fold(onSuccess = { it.keyId!! }, onFailure = { return KmmResult.failure(it) })
```

Comment:
What if `keyId` is null?

Reply:
This should still be identifier. Thanks

Reply:
tricky one. this issue was always there, but hidden away